app/connectors/des/DesConnector.scala

```diff
@@ -52,32 +57,44 @@ class DesConnector @Inject() (servicesConfig: ServicesConfig, httpClient: HttpCl
     val now = LocalDate.now()
     val aYearAgo = LocalDate.now().minusYears(1)
     implicit val hc: HeaderCarrier = desHeaderCarrier
-    val getFinancialURL: String = s"$serviceURL$financialsUrl/${vrn.value}/VATC?dateFrom=${aYearAgo.toString}&dateTo=${now.toString}"
-    logger.debug(s"""Calling des api 1166 with url $getFinancialURL""")
-    httpClient.GET[Option[FinancialData]](getFinancialURL, headers = headers)
+    val getFinancialURL = new URL(s"$serviceURL$financialsUrl/${vrn.value}/VATC?dateFrom=${aYearAgo.toString}&dateTo=${now.toString}")
```

Comment:
looks like 
`new URL() ` 
is deprecated, could we not use just url interpolator, something like:
`    val getCustomerURL = url"$serviceURL$customerUrl/${vrn.value}/information"
`

Reply:
it was doing something funny where it would drop the port number in the url in the tests - I couldn't figure out why so just changed it to `new URL` and that seemed to make it happier. I can take a look again to see if I can figure it out

Reply:
Starting Java 20: https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URL.html#constructor-deprecation